server/router/router.go

```diff
@@ -17,7 +18,7 @@ func NewRouter () *mux.Router {
 				Methods(route.Method).
 				Path(route.Pattern).
 				Name(route.Name).
-				Handler(handler)
+				Handler(middleware.AddDefaultCORSHeaders(handler))
```

Comment:
this adds the CORS headers, did you remove the CORs middleware from the top level?

Reply:
i removed the cors package i originally brought in before coding my own middleware. there is nothing on main.go for CORS anymore.